pandas/core/internals/blocks.py

```diff
@@ -925,26 +925,28 @@ def putmask(
         inplace: bool = False,
         axis: int = 0,
         transpose: bool = False,
-    ):
+    ) -> List["Block"]:
         """
         putmask the data to the block; it is possible that we may create a
         new dtype of block
 
-        return the resulting block(s)
+        Return the resulting blocks.
 
         Parameters
         ----------
-        mask  : the condition to respect
+        mask : the condition to respect
```

Comment:
regarding "Parameter "mask" has no description", "Numpydoc doesn't require types ... it's only our internal validation that requires this. We could perfectly well decide to relax this" https://github.com/pandas-dev/pandas/issues/32773#issuecomment-600284371 
```suggestion
        mask
            The condition to respect.
```

Reply:
I left this one as-is because I have a separate branch coming up that allows us to type mask as ndarray[bool]